AutomatedTesting/Gem/PythonTests/Atom/TestSuite_Benchmark_GPU.py

```diff
@@ -15,16 +16,76 @@
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.parametrize('rhi', ['dx12', 'vulkan'])
+def filebeat_service_running():
+    """
+    Checks if the filebeat service is currently running on the OS.
+    :return: True if filebeat service detected, False otherwise.
+    """
+    result = True
+    try:
+        filebeat_service = psutil.win_service_get('filebeat')
+        filebeat_service_info = filebeat_service.as_dict()
+        if filebeat_service_info['status'] == 'stopped':
```

Comment:
is there more than one not running status? should this be inverted to
result = False
if 'status' == 'running'
    result = True